lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -81,6 +82,10 @@ export default function ContentSelector({
         pagesEnabled: canvasPagesEnabled,
         listPages: listPagesApi,
       },
+      canvasStudio: {
+        enabled: canvasStudioEnabled,
+        listMedia: listCanvasStudioMediaApi,
```

Comment:
This follows the pattern of other content sources, although I think we should consider unifying all the content sources that use our file picker, to reduce the near-duplicated code that needs to be maintained.